disnake/channel.py

```diff
@@ -5,18 +5,8 @@
 import asyncio
 import datetime
 import time
-from collections.abc import Iterable, Mapping, Sequence
-from typing import (
-    TYPE_CHECKING,
-    Any,
-    Callable,
-    Literal,
-    NamedTuple,
-    Optional,
-    Union,
-    cast,
-    overload,
-)
+from collections.abc import Callable, Iterable, Mapping, Sequence
+from typing import TYPE_CHECKING, Any, Literal, NamedTuple, Optional, Union, cast, overload
```

Comment:
I had the same idea with compacting the typing imports (and then force-pushed it away again), iirc the final verdict was waiting for the 3.10 change as that'll get rid of even more typing imports
<img width="923" height="139" alt="image" src="https://github.com/user-attachments/assets/e130a1c5-0bb2-4e2a-8dc7-e5680175abee" />